scripts/microupload.py

```diff
@@ -68,7 +68,7 @@ def main(args: List[str]) -> None:
     created_cache = set()
     for path in progress('Uploading files', to_upload):
         local_path = os.path.abspath(path)
-        remote_path = os.path.normpath(path)
+        remote_path = os.path.normpath(path).replace(os.path.sep, '/')
```

Comment:
I like that `remote_path` is updated before we print the message about uploading a file in the verbose mode.

Reply:
I agree that is a better place to make this since it will be visible in the console.